playground/backend/internal/environment/environment_service.go

```diff
@@ -178,10 +178,12 @@ func ConfigureBeamEnvs(workDir string) (*BeamEnvs, error) {
 			sdk = pb.Sdk_SDK_PYTHON
 		case pb.Sdk_SDK_SCIO.String():
 			sdk = pb.Sdk_SDK_SCIO
+		default:
+			return nil, errors.New("incorrect value of sdk in the environment")
 		}
 	}
 	if sdk == pb.Sdk_SDK_UNSPECIFIED {
-		return nil, errors.New("env BEAM_SDK must be specified in the environment variables")
+		return NewBeamEnvs(sdk, nil, preparedModDir, numOfParallelJobs), nil
```

Comment:
It seems that if SDK is Unspecified then we receive `nil, errors.New("incorrect value of SDK in the environment")` on the `182` line, so code on the `185` line doesn't require.
Maybe we need to remove `default` case?